Rworkshops.html

```diff
@@ -71,16 +71,16 @@ <h1 id="logo"><a href="index.html"><span>databrew</span></a></h1>
 								<div class="6u 12u(narrower)">
 
 									<section>
-										<a href="#" class="image featured"><img src="images/workshop.jpg" alt="" /></a>
+										<a href="#" class="image featured"><img src="images/0.jpg" alt="" /></a>
 										<header>
-											<h3>R workshop 2</h3>
+											<h3>Introduction to R for health research</h3>
```

Comment:
Small detail, but I'd capitalize 'Health' and 'Research' to make it consistent with Ben's course title.